src/tests/unit/tests/reports/library/report-html-generator-initializer.test.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { reportHtmlGeneratorInitializer } from "reports/library/report-html-generator-initializer";
+import { ReportHtmlGenerator } from "reports/report-html-generator";
+
+describe('ReportHtmlGeneratorInitializer', () => {
+
+    it('returns a valid generator', () => {
+        const initalizer = reportHtmlGeneratorInitializer();
+
+        expect(initalizer).toBeInstanceOf(ReportHtmlGenerator);
```

Comment:
Is there an advantage to using toBeInstanceOf vs. notToBeNull?

Reply:
It validates that it returns an object of the correct type. Just a little more rigorous.